src/components/Settings/SettingsPage.vue

```diff
@@ -262,6 +320,43 @@
         </el-button>
       </span>
     </el-dialog>
+    <el-dialog
+      data-cy="addWalletLimit"
+      title="Add wallet limit"
+      :visible.sync="addWalletLimitDialogVisible"
+      :show-close="false"
+      width="450px"
+      center
+    >
+      <div class="approval_form-desc">
+        <el-form class="dialog-form">
+          <el-form-item label="Wallets">
+            <el-select class="fullwidth">
+              <el-option
+                v-for="wallet in wallets"
+                :key="wallet.name"
+                :label="`${wallet.name} (${wallet.asset.toUpperCase()})`"
+                :value="wallet.assetId">
+              </el-option>
+            </el-select>
+          </el-form-item>
+          <el-form-item label="Limit">
+            <el-input placeholder="0"/>
```

Comment:
It would be nice to add type="number" to prevent entering of non-number value. We can add it to every number input in project